src/shared/constants.js

```diff
@@ -65,10 +78,75 @@ export const USER_ROLES = {
 export const PAGES_KEYS = {
   tasks: 'tasks',
   progress: 'progress',
+  track: 'tracks',
 };
 
 export const STATUS_KEYS = {
   active: 'Active',
   success: 'Success',
   fail: 'Fail',
 };
+
+export const LINKPATH_KEYS = {
+  track: 'track',
+  progress: 'progress',
+  tasks: 'tasks',
+};
+
+export const MODALTITLE_KEYS = {
+  createMember: 'Create member',
+  createTask: 'Create task',
+  createTrack: 'Create track',
+};
+
+export const USER_FIELDS_KEYS = [
+  { name: 'name', title: 'Name:', type: 'text', required: true },
+  { name: 'lastName', title: 'Last name:', type: 'text', required: false },
+  { name: 'email', title: 'Email:', type: 'text', required: true },
+  { name: 'direction', title: 'Direction:', type: 'select', options: DIRECTIONS_OPTIONS_KEYS, required: false },
+  { name: 'sex', title: 'Sex:', type: 'select', options: SEX_OPTIONS_KEYS, required: true },
+  { name: 'role', title: 'Role:', type: 'select', options: ROLE_OPTIONS_KEYS, required: true },
+  { name: 'password', title: 'Password:', type: 'password', required: true },
+  { name: 'confirmPassword', title: 'Confirm password:', type: 'password', required: true },
+  { name: 'birthDate', title: 'Date of Birth:', type: 'date', required: true },
+  { name: 'address', title: 'Address:', type: 'text', required: true },
+  { name: 'phone', title: 'Mobile phone:', type: 'text', required: true },
+  { name: 'skype', title: 'Skype:', type: 'text' },
+  { name: 'startDate', title: 'Start date:', type: 'date', required: true },
+  { name: 'education', title: 'Education:', type: 'text', required: true },
+  { name: 'universityAverageAcore', title: 'University average score:', type: 'text', required: true },
+  { name: 'mathScore', title: 'Math score:', type: 'text', required: true },
+];
+
+export const TASK_FIELDS_KEYS = [
+  { name: 'name', title: 'Name:', type: 'text', required: true },
+  { name: 'description', title: 'Description:', type: 'text', required: false },
+  { name: 'startDate', title: 'Start date:', type: 'date', required: true },
+  { name: 'deadlineDate', title: 'Deadline:', type: 'date', required: true },
+];
+
+export const TRACK_FIELDS_KEYS = [
+  { name: 'name', title: 'Name:', type: 'select', required: true },
+  { name: 'date', title: 'Date:', type: 'date', required: true },
+  { name: 'node', title: 'Node:', type: 'text', required: true },
+];
+
+export const FOOTER_KEYS = {
+  copyRight: ' CopyRight: Evgeniy Moroz',
+  years: '2022',
```

Comment:
This value changes every year. I've already mentioned it twice, you should generate it dynamically

Reply:
```suggestion
export const FOOTER_KEYS = {
  creator: 'Evgeniy Moroz',
  years: `${getCurrentYear()}`,
};
```